tests/scripts/run-throughput-perf.py

```diff
@@ -32,31 +32,20 @@
 # Globals
 ##########################################################################
 
-# List of dlls we want to crossgen
-dll_list = {
+# List of dlls we want to exclude
+dll_exclude_list = {
     'Windows_NT': [
-        "System.Private.CoreLib",
-        "System.Reflection.Metadata",
-        "System.Linq.Expressions",
-        "Microsoft.CSharp",
-        "System",
-        "Microsoft.CodeAnalysis.VisualBasic",
-        "System.Private.DataContractSerialization",
-        "System.Core",
-        "System.Xml",
-        "Microsoft.CodeAnalysis.CSharp",
-        "Microsoft.CodeAnalysis",
-        "System.Linq.Parallel",
-        "System.Private.Xml"
+        # Require Newtonsoft.Json
```

Comment:
What does this comment mean?  Also how did you decide on what to put in the exclude list?